utils/stripe_utils.py

```diff
@@ -55,6 +59,10 @@ class StripeUtils(object):
     CURRENCY = 'chf'
     INTERVAL = 'month'
```

Comment:
INTERVAL=month might or might not be a good idea. On one hand it is easy for the user, but on the other hand it will require to handle partial payments. I suggest to use 30 days instead. This will also open up easy calculations for the future, when we will allow booking for hours only.